app/model/DigitalEdition.scala

```diff
@@ -0,0 +1,25 @@
+package model
+
+import configuration.Config
+
+case class DigitalEdition(id: String, name: String, price: String, campaign: String)
+
+
+object DigitalEdition {
+
+  trait UrlConfig {
+    val externalSubscriptionUrl: String
+  }
+
+  object UK extends DigitalEdition("uk", "UK", "£11.99", "dis_2408")
+  object US extends DigitalEdition("us", "US", "$19.99", "dis_2378")
+  object AU extends DigitalEdition("au", "Australia", "$21.50", "dis_2379")
+  implicit lazy val defaultUrlConfig: UrlConfig = Config
+
+  def getRedirect(edition: DigitalEdition)(implicit urlConfig: UrlConfig): String = {
```

Comment:
Just to follow up on `java.net.URL` - unfortunately it's quite a **bad** class, see  http://www.eishay.com/2008/04/javas-url-little-secret.html or
http://michaelscharf.blogspot.co.uk/2006/11/javaneturlequals-and-hashcode-make.html

Because of this, `java.net.URI` is preferable - or the representation in https://github.com/NET-A-PORTER/scala-uri , which we already use elsewhere.